src/data/skill-desc.csv

```diff
@@ -383,13 +383,13 @@
 "すぐ後ろにウマ娘が長くいると\n速度がわずかに上がる","<size=22>Slightly increase speed when a girl has been following you for a while</size>"
 "レース中盤に競り合うと\n速度が上がる＜ダート＞","<size=22>Dirt・Increase speed when competing for position during the Middle Leg</size>"
 "レース中盤に競り合うと\n速度がわずかに上がる＜ダート＞","<size=22>Dirt・Slightly increase speed when competing for position during the Middle Leg</size>"
-"レースへの渇望により\n能力が少し発揮しやすくなる",""
-"かつての走りとは違うがあの頃と\n変わらない想いが能力をすごく上げる",""
+"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to become able to demonstrate your power a little better</size>"
```

Comment:
```suggestion
"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to be able to demonstrate your power a little better</size>"
```

Reply:
Alternatively
```suggestion
"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to demonstrate your power a little better</size>"
```

Reply:
Going to apply this suggestion but replace power with ability.